initial-data-load/src/mapLayers/db.js

```diff
@@ -10,8 +10,10 @@ const createIndex = async () => {
   await mongoose.connection.db.collection('mapLayers').createIndex({ title: 1 });
 };
 
-const createGeoDataIndex = async (collectionName) => {
-  await mongoose.connection.db.collection(collectionName).createIndex({ geometry: '2dsphere' });
+const createGeoDataIndex = async (collectionName, geoKey = 'geometry') => {
+  const indexKey = {};
+  indexKey[geoKey] = '2dsphere';
+  await mongoose.connection.db.collection(collectionName).createIndex(indexKey);
 };
 
 const deleteAllFromCollection = async (collectionName) => {
```

Comment:
There is warning in log:
initial-data-load-service_1  | (node:29) DeprecationWarning: collection.remove is deprecated. Use deleteOne, deleteMany, or bulkWrite instead.

maybe you can fix it in this task or leave it for fixing later.

Reply:
I would like to investigate the difference between 'remove' and the replacement before using one, especially in context of compatibility with CosmoDB. Definitely out of scope here